numpy/doc/basics.py

```diff
@@ -155,10 +155,10 @@
 ``long double`` identical to ``double`` (64 bits). NumPy makes the
 compiler's ``long double`` available as ``np.longdouble`` (and
 ``np.clongdouble`` for the complex numbers). You can find out what your
-numpy provides with ``np.finfo(np.longdouble)``\.
+numpy provides with ``np.finfo(np.longdouble)``.
 
 NumPy does not provide a dtype with more precision than C
-``long double``\s; in particular, the 128-bit IEEE quad precision
+``long double``s; in particular, the 128-bit IEEE quad precision
```

Comment:
This one is correct - the one on the line below (which has always been there) was wrong. Sorry if I was unclear.